gcp/workers/importer/importer.py

```diff
@@ -816,6 +828,8 @@ def _process_deletions_bucket(self,
       logging.error(
           'Cowardly refusing to delete %d missing records from '
           'GCS for: %s', len(vulns_to_delete), source_repo.name)
+      vulns = [v.id for v in vulns_to_delete]
+      logging.error('Vulnerabilities to delete: %s', vulns)
```

Comment:
Make this a `logging.info` so we don't trigger alerts for both logs.
Or, just add it to the `json_fields` of the previous log.